client/state/connectionMiddleware/middleware.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+const connections = {};
+
+export const connectionMiddleware = store => next => action => {
+	if ( connections.hasOwnProperty( action.type ) ) {
+		connections[ action.type ].connectionFunction( wpcom )( action, store.dispatch );
+		next( action );
+	} else {
+		next( action );
+	}
```

Comment:
Yes, this is an artifact because I cut out the queuing methods. I will remove it once we decide to move with that PR